### PR TITLE
Added doc-lists notifications

### PR DESCRIPTION
**Pull request by saundefined**
Also, enable notifications for:

`doc-{LANG}` to doc-{LANG}@lists.php.net
`doc-base` to phpdoc@lists.php.net

### CODE REVIEW

File: github-webhook.php [hunk 1]
```diff
@@ -104,6 +104,8 @@ function handle_commit($commit) {
         'php-src' => 'git-pulls@lists.php.net',
         'web-' => 'php-webmaster@lists.php.net',
         'pecl-' => 'pecl-dev@lists.php.net',
+        'doc-' => '#repository#@lists.php.net',
+        'doc-base' => 'phpdoc@lists.php.net',
```
Comment (nikic) [inline github-webhook.php:108]:
```suggestion
        'doc-base' => 'doc-cvs@lists.php.net',
```

**Comment by nikic** [comment]
Note that these are pull request mails, not commit mails. Do people really want to receive them on the mailing lists?

**Comment by nikic** [comment]
I believe php-src and php-langspec are the only ones here that were active historically (because other repos don't have the webhook for that registered).

**Comment by saundefined** [comment]
@nikic, thanks!

> Note that these are pull request mails, not commit mails. Do people really want to receive them on the mailing lists?

Hmm ... earlier when posting a patch to the doc-repo (from the PHP Doc editor), we received an email notification.

If you think these notifications are redundant (and GitHub notifications are sufficient), we can close the PR.

**Comment by nikic** [comment]
Maybe @cmb69 can chime in? Should we send out mails for doc-* PRs?

**Comment by cmb69** [comment]
IMO, that's not necessary, and may flood the respective mailing list. I prefer to subscribe to the repos I'm interested in (and to unsubscribe from individual PRs).

**Comment by saundefined** [comment]
@nikic @cmb69, thanks, then I close it

**Pull request closed without merging**